src/python/pants/backend/jvm/tasks/jvm_compile/rsc/rsc_compile.py

```diff
@@ -176,6 +176,10 @@ def register_options(cls, register):
       ]
     )
 
+  @classmethod
+  def product_types(cls):
+    return ['runtime_classpath', 'rsc_mixed_compile_classpath', 'zinc_analysis', 'zinc_args']
```

Comment:
This should use `super(..).product_types() + ...` rather than redefining them from scratch.